src/arsenic/utils.py

```diff
@@ -10,14 +11,16 @@ def free_port() -> int:
         return sock.getsockname()[1]
 
 
-def px_to_int(value: str) -> int:
+def px_to_number(value: str) -> Union[int, float]:
     original = value
     if value.endswith("px"):
         value = value[:-2]
     if value.isdigit():
         return int(value)
-    else:
-        raise ValueError(f"{original!r} is not an int or <int>px value")
+    try:
+        return float(value)
+    except:
```

Comment:
I'd recommend `except Exception` for principle of least astonishment, or in fact `ValueError` because the argument is known to be a `str`.